src/components/input-time-picker/input-time-picker.tsx

```diff
@@ -449,6 +500,11 @@ export class InputTimePicker
     }
   };
 
+  private restoreInputFocus(): void {
```

Comment:
I think you can drop this method and call `calciteInputEl.setFocus()` where needed instead.